HPXMLtoOpenStudio/resources/waterheater.rb

```diff
@@ -88,14 +90,19 @@ def self.apply_heatpump(model, runner, loc_space, loc_schedule, weather, water_h
         Schedule.set_schedule_type_limits(model, setpoint_schedule, Constants.ScheduleTypeLimitsTemperature)
 
         # Actuated schedule
-        control_setpoint_schedule = OpenStudio::Model::ScheduleConstant.new(model)
-        control_setpoint_schedule.setName("#{obj_name_hpwh} ControlSetpoint")
+        if power_outage_periods.empty?
+          # need to avoid Always Off Discrete "because it has an incompatible ScheduleTypeLimits"
+          control_setpoint_schedule = OpenStudio::Model::ScheduleConstant.new(model)
+          control_setpoint_schedule.setName("#{obj_name_hpwh} ControlSetpoint")
```

Comment:
I put this in because of updates/changes right around here: https://github.com/NREL/OpenStudio-HPXML/blob/power-outage2/HPXMLtoOpenStudio/resources/schedules.rb#L28. If you tried to use the "Always Off Discrete" schedule, you'd get a incompatible ScheduleTypeLimits error. It appears the additional criteria I add here to fall into this block makes this change unnecessary however... I'll revert.